db/migrate/20230913011741_create_exchanges.rb

```diff
@@ -0,0 +1,11 @@
+class CreateExchanges < ActiveRecord::Migration[7.0]
+  def change
+    create_table :exchanges do |t|
+      t.decimal :amount
+      t.string :name
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [Optional] Database schema was supposed to reflect the [structure given](https://gitlab.com/microverse/guides/tse/code_review/code_review_guidelines_2_0/-/raw/master/ror/capstone/images/erd_diagram.png) to you. Instead of the default `user_id` the the ERD for this project shows `author_id`